.gitignore

```diff
@@ -19,3 +19,4 @@ Session.vim
 .netrwhist
 **/charts/*.tgz
 .history
+values.yaml
```

Comment:
Tonic does not do defaults in `values.yaml`, so we deleted and re-ignored. The defaults are implemented instead in more complex templates.